Makefile

```diff
@@ -73,9 +121,24 @@ help: ## Display this help.
 
 ##@ Development
 
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+manifests: controller-gen dependencies-manifests ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
+.PHONY: dependencies-manifests
+dependencies-manifests: export KUADRANT_CONTROLLER_GITREF := $(KUADRANT_CONTROLLER_GITREF)
+dependencies-manifests: export AUTHORINO_OPERATOR_GITREF := $(AUTHORINO_OPERATOR_GITREF)
+dependencies-manifests: export LIMITADOR_OPERATOR_GITREF := $(LIMITADOR_OPERATOR_GITREF)
+dependencies-manifests: ## Update kuadrant dependencies manifests.
+	envsubst \
+        < config/dependencies/controller/kustomization.template.yaml \
+        > config/dependencies/controller/kustomization.yaml
+	envsubst \
+        < config/dependencies/authorino/kustomization.template.yaml \
+        > config/dependencies/authorino/kustomization.yaml
+	envsubst \
+        < config/dependencies/limitador/kustomization.template.yaml \
+        > config/dependencies/limitador/kustomization.yaml
```

Comment:
I couldn't figure out a better way of templating the kustomization file, but the point of this is to make this Makefile the source of truth for all kuadrant component versions, and allow you to override the version as required. Also not sure if `envsubst` works on macOS.

To install the latest authorino operator for example:

```
make dependencies-manifests AUTHORINO_OPERATOR_VERSION=latest
./bin/kustomize build config/dependencies/authorino | kubectl apply -f -
```